evm/v0.5/src/helpers.ts

```diff
@@ -636,6 +857,36 @@ export function eventArgs(event?: ethers.Event) {
   return (event?.args as any) as EventArgsArray
 }
 
+/**
+ * Extract array of logs from a transaction
+ * @param tx The transaction to wait for, then extract logs from
+ */
+export async function getLogs(
```

Comment:
I'm not sure how I feel about the log helper functions, because they consume the `receipt` variable, you aren't able to access them afterwards if you need to use the `receipt` variable. This makes the function specific to the use case where you want to only get logs from a transaction and don't care about any other properties about it. I'm not sure if this is an assumption we want to make.

Reply:
Another reason for why we may not need this is that often enough we don't need to get logs directly. If the log we're interested is being emitted from the contract method we called via ethers, we can get the event by name via the `events` property of the `ContractReceipt`